### PR TITLE
[4.0] Use Web Asset manager for bootstrap

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue # .

### Summary of Changes

Define and properly use the javascript files for Bootstrap

### Testing Instructions

Needs `npm install`. Make sure that all interactive Bootstrap components are still functional 

### Documentation Changes Required

No, not for this one

### CODE REVIEW

File: libraries/src/HTML/Helpers/Bootstrap.php [hunk 1]
```diff
@@ -72,6 +69,13 @@ public static function alert($selector = '.alert') :void
 			$options = array_merge($scriptOptions, $options);
 		}
 
+		// Include the Bootstrap component
+		Factory::getApplication()
+			->getDocument()
+			->getWebAssetManager()
+			->useScript('bootstrap.alert')
+			->useScript('bootstrap.es5');
```
Comment (Fedik) [inline libraries/src/HTML/Helpers/Bootstrap.php]: it enough to call `->useScript('bootstrap.alert')`, and `bootstrap.es5` will be loaded automatically as dependency

Comment (dgrammatiko) [reply libraries/src/HTML/Helpers/Bootstrap.php]: Yeah just realised that, but already fixed: https://github.com/joomla/joomla-cms/pull/32208/commits/64c9a64ad7c55c63fa787ff15325063df15afd89

**Comment by Fedik** [comment]
I have tested this item :white_check_mark: successfully on 64c9a64ad7c55c63fa787ff15325063df15afd89<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32208">issues.joomla.org/tracker/joomla-cms/32208</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by wilsonge** [comment]
@dgrammatiko nope. just realised this breaks the framework method that should load all scripts. Please can you do a fix for that one

**Comment by Fedik** [comment]
@wilsonge here #32217

**Pull request merged**